### DISCUSSION TITLE
Monitoring multiple meters

### DISCUSSION THREAD

**Discussion by hyteckit** [bemasher/rtlamr#200] [Q&A] [answered]
Haven't been following this development since 2017. I remember modifying the scripts to monitor multiple electric meters since I have 3 separate electric meters.

Does it now monitor multiple meters and record data to influxdb for multiple meters? Looks like rtlamr-collect is what's used for data aggregation. Sounds like from the README, it now handles multiple meters and records data for all the meters it detects unless you use "RTLAMR_FILTERID"?

**Comment by bemasher** [comment, marked as answer]
Without either `-filterid` or `RTLAMR_FILTERID`, rtlamr will decode and report all valid messages. With rtlamr-collect it is recommended to specify a set of meters to listen to because influxdb does not gracefully handle high series cardinality that would result from listening without a filter.
